tests/countries/test_nepal.py

```diff
@@ -0,0 +1,536 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import PUBLIC, WORKDAY
+from holidays.countries.nepal import Nepal, NP, NPL
+from tests.common import CommonCountryTests
+
+
+class TestNepal(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(2011, 2050)
```

Comment:
If the `start_year` is 2010, shouldn't this be 2010 too?
```suggestion
        years = range(2010, 2050)
```

Remember to update the start year for all other test cases as well 👍

Reply:
Yes, agreed! I was getting an error in the ranged holiday assertions starting from 2010—likely because I missed updating this in setUpClass. I’ll make all the necessary changes right away.